tests/e2e/specs/login-localize-script.test.js

```diff
@@ -10,41 +10,50 @@ import fs from 'fs';
 import path from 'node:path';
 
 test.describe( 'Localize Script on wp-login.php', () => {
-	const muFolder = path.normalize(
+	const srcMuFolder = path.normalize(
 		path.join( process.cwd(), 'src/wp-content/mu-plugins' )
 	);
-	const muFile = path.normalize( path.join( muFolder, 'login-test.php' ) );
+	const srcMuFile = path.normalize( path.join( srcMuFolder, 'login-test.php' ) );
+	const buildMuFolder = path.normalize(
+		path.join( process.cwd(), 'build/wp-content/mu-plugins' )
+	);
+	const buildMuFile = path.normalize( path.join( buildMuFolder, 'login-test.php' ) );
```

Comment:
IMHO it's a bit wasteful and overkill to do this in both folders by default. To a maintainer it's not clear why it's added in both places, just hoping that one of them works.

We should just look at the `process.env.LOCAL_DIR` variable instead.

In `tests/e2e/playwright.config.js` we can default it to `src` if it's not set already. In GitHub Actions it's set to `build`, so it would automatically pick that location.